src/odemis/gui/comp/overlay/cryo_feature.py

```diff
@@ -267,3 +306,28 @@ def set_icon(feature_icon):
                 self._label.draw(ctx)
 
             ctx.paint()
+
+    def _view_to_stage_pos(self, v_pos):
+        """Convert view position to stage position"""
+        p_pos = self.cnvs.view_to_phys(v_pos, self.cnvs.get_half_buffer_size())
+        new_pos = {
+            "x": p_pos[0],
+            "y": p_pos[1],
+            "z": self.tab_data.main.stage.position.value["z"],  #NOTE: we cannot use cnvs.view._stage, because the acquired cnvs does not have a _stage....? why?
+        }
+        pos = self.pm.from_sample_stage_to_stage_position(new_pos)
+        return pos
+
+
+    def _get_feature_position_at_view_posture(self, feature: CryoFeature) -> Dict[str, float]:
+        """Get the feature position at the view posture, create it if it doesn't exist"""
+
+        return get_feature_position_at_posture(
+            pm=self.pm,
+            feature=feature,
+            posture=self.view_posture,
+        )
+
+    def _on_view_posture_change(self, posture):
+        self.view_posture = posture
+        self.cnvs.update_drawing()
```

Comment:
empty line at the end

Reply:
same here, one empty line needs to be added at the end of the file according to the coding guidelines

Reply:
Isn't it just a single empty line at the end? or is it two because its a class?

Reply:
There is just one single line at the end whether it is a class function or not. I see that the last line 333 is self.cnvs.update_drawing().
```suggestion
        self.cnvs.update_drawing()
        
```